code/modules/power/generators/turbine.dm

```diff
@@ -200,12 +243,11 @@
 		return
 	. += image(icon, "comp-o[rpm_threshold]", FLY_LAYER)
 
-// These are crucial to working of a turbine - the stats modify the power output. TurbGenQ modifies how much raw energy can you get from
-// rpms, TurbGenG modifies the shape of the curve - the lower the value the less straight the curve is.
+// These are crucial to working of a turbine - the stats modify the power output. TurbPower modifies how much raw energy can you get from
+// rpms, TURBCURVESHAPE modifies the shape of the curve - the lower the value the less straight the curve is.
 
-#define TURBPRES 9000000
-#define TURBGENQ 100000
-#define TURBGENG 0.5
+#define TURBPOWER 500000
+#define TURBCURVESHAPE 0.5
 
```

Comment:
These will need to be rewritten even more with some examples - act like the reader only knows the basics of thermodynamica, I'll be glad to help you out in the next weekend!

Reply:
Yeah this would be nice to have.